app/views/shared/_top_navbar.html.erb

```diff
@@ -1,19 +1,21 @@
 <nav id="topnav" class="navbar navbar-expand-always container" role="navigation">
```

Comment:
So a double `container`?

Reply:
Thanks for the catch; no, not a double one, just putting one where bootstrap expects it to be.